### PR TITLE
[WIP] Refresh Github Welcome page and add auto labeler Action

### PR DESCRIPTION
**Pull request by jhillairet**
- Refresh the welcome page (reorder badges, code description before install, community messages)
- Add the automatic [pull request labeler]((https://github.com/actions/labeler) to GitHub Actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +14,45 @@ in the Python programming language. It provides a modern, object-oriented librar
...
+
+## Questions or comments? Ask the Community
+
+[![Mailing List](https://img.shields.io/badge/mail-Mailing%20list-brightgreen)](http://groups.google.com/group/scikit-rf)
+[![Slack](https://img.shields.io/badge/Slack-join%20chat-blueviolet?style=flat&logo=slack)](https://scikit-rf.slack.com)
+[![Matrix](https://img.shields.io/badge/Matrix-join%20chat-blueviolet?style=flat&logo=matrix)](https://app.element.io/#/room/#scikit-rf:matrix.org)
+
+* [Mailing list](http://groups.google.com/group/scikit-rf): You can suscribe to our mailing list to ask (and answer) questions and to receive scikit-rf annoucements.
+* Chat: scikit-rf have channels on both [Slack](https://scikit-rf.slack.com) and [Matrix](https://app.element.io/#/room/#scikit-rf:matrix.org)  
+* [Suggestion box](https://docs.google.com/forms/d/e/1FAIpQLSfkSNy8KzoHwdDxYGCmHHiA3GsHgN6zKTFI2NrcUR29U7Xx6A/viewform?usp=sf_link): if you would like to (optionally anonymoulsy) suggest a feature/topic for consideration.
```
Comment (JAnderson419) [inline README.md:47]: typo here. I would also recommend the following wording:

> if you would like to suggest a feature/topic for consideration (optionally anonymous).

File: README.md [hunk 2]
```diff
@@ -21,13 +14,45 @@ in the Python programming language. It provides a modern, object-oriented librar
...
+conda install -c conda-forge scikit-rf 
+```
+
+## Questions or comments? Ask the Community
+
+[![Mailing List](https://img.shields.io/badge/mail-Mailing%20list-brightgreen)](http://groups.google.com/group/scikit-rf)
+[![Slack](https://img.shields.io/badge/Slack-join%20chat-blueviolet?style=flat&logo=slack)](https://scikit-rf.slack.com)
+[![Matrix](https://img.shields.io/badge/Matrix-join%20chat-blueviolet?style=flat&logo=matrix)](https://app.element.io/#/room/#scikit-rf:matrix.org)
+
+* [Mailing list](http://groups.google.com/group/scikit-rf): You can suscribe to our mailing list to ask (and answer) questions and to receive scikit-rf annoucements.
```
Comment (JAnderson419) [inline README.md:45]: typo in announcements

File: README.md [hunk 3]
```diff
@@ -21,13 +14,45 @@ in the Python programming language. It provides a modern, object-oriented librar
...
+* [Documentation](http://scikit-rf.readthedocs.org/)
+
+## Install
 
+[![PyPI version](https://img.shields.io/pypi/v/scikit-rf?style=flat&logo=pypi)](https://pypi.org/project/scikit-rf/)
+[![Conda-forge version](https://img.shields.io/conda/v/conda-forge/scikit-rf?style=flat&logo=anaconda)](https://img.shields.io/conda/v/conda-forge/scikit-rf)
+[![Conda version](https://anaconda.org/conda-forge/scikit-rf/badges/latest_release_date.svg)](https://anaconda.org/conda-forge/scikit-rf)
 
+If you have [installed Python](https://wiki.python.org/moin/BeginnersGuide/Download),
+you can install PlasmaPy from [pip](https://pypi.org/project/pip/)
```
Comment (JAnderson419) [inline README.md:26]: PlasmaPy -> skrf

File: README.md [hunk 4]
```diff
@@ -21,13 +14,45 @@ in the Python programming language. It provides a modern, object-oriented librar
...
+via
+```Shell
+python -m pip install scikit-rf
+```
+
+If you have
+[installed conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html),
+then you can also get PlasmaPy from
```
Comment (JAnderson419) [inline README.md:34]: PlasmaPy -> skrf

**Review by JAnderson419 — COMMENTED** [review]
Pointed out a few typos in Readme. Looks great overall!

File: README.md (hunk 1 above)
Comment (jhillairet) [reply README.md:47]: OK

**Comment by jhillairet** [comment]
Here is a preview of the README page
image

**Comment by jhillairet** [comment]
@arsenovic , @JAnderson419 and others, do you think of additional things to highlight for the main page? Or different ways to do it?

**Comment by arsenovic** [comment]
i like it !

**Pull request merged**